sys/openbsd/dev_bpf.txt.const

```diff
@@ -27,5 +27,3 @@ BIOCSRTIMEOUT = amd64:2148549229
 BIOCVERSION = amd64:1074020977
 IFNAMSIZ = amd64:16
 IFREQ_SIZE = amd64:32
-SYS_ioctl = amd64:54
```

Comment:
TODO: why it disappeared?

Reply:
It's WAI: https://github.com/google/syzkaller/commit/59e959932704c7a500e97fda11bbae7ccca50020